client/payment-details/dispute-details/dispute-steps.tsx

```diff
@@ -0,0 +1,189 @@
+/** @format **/
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { __, _n, sprintf } from '@wordpress/i18n';
+import { createInterpolateElement } from '@wordpress/element';
+import { dateI18n } from '@wordpress/date';
+import moment from 'moment';
+import HelpOutlineIcon from 'gridicons/dist/help-outline';
+import classNames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import type { Dispute } from 'wcpay/types/disputes';
+import { ChargeBillingDetails } from 'wcpay/types/charges';
+import { formatExplicitCurrency } from 'utils/currency';
+import { ClickTooltip } from 'wcpay/components/tooltip';
+
+interface Props {
+	dispute: Dispute;
+	customer: ChargeBillingDetails | null;
+	chargeCreated: number;
+	daysRemaining: number;
+}
+
+const DisputeSteps: React.FC< Props > = ( {
+	dispute,
+	customer,
+	chargeCreated,
+	daysRemaining,
+} ) => {
+	let emailLink;
+	if ( customer?.email ) {
+		const chargeDate = dateI18n(
+			'Y-m-d',
+			moment( chargeCreated * 1000 ).toISOString()
+		);
+		const disputeDate = dateI18n(
+			'Y-m-d',
+			moment( dispute.created * 1000 ).toISOString()
+		);
+		const emailSubject = `Problem with your purchase from ${ wcpaySettings.storeName } on ${ chargeDate }?`;
+		const emailBody =
+			`Hello ${ customer?.name }\n\n` +
+			`We noticed that on ${ disputeDate }, you disputed a ${ formatExplicitCurrency(
+				dispute.amount,
+				dispute.currency
+			) } from ${ chargeDate }. We wanted to contact you to make sure everything was all right with your purchase and see if there's anything else we can do to resolve any problems you might have had.\n\n` +
+			`Alternatively, if the dispute was a mistake, you could easily withdraw it by calling the number on the back of your card. Thank you so much - we appreciate your business and look forward to working with you.`;
+
+		emailLink = `mailto:${ customer.email }?subject=${ encodeURIComponent(
```

Comment:
Wanted to use [querystring.strinfigy()](https://nodejs.org/api/querystring.html#querystringstringifyobj-sep-eq-options) but I don't see it's used at all in the codebase.

Reply:
Has anyone tested clicking the link correctly works with at least one email client? If there's any issue with escaping or protocol then that would catch it! I have this behaviour disabled on my mac (because I use GMail), so wasn't able to test. 

Noting so we can test in our test swarm today e.g. on mobile or if someone has `mailto` configured right :)